app/controllers/collections_controller.rb

```diff
@@ -202,7 +231,7 @@ def delete_document_from_item
 
   def edit_collection
     begin
-      collection = validate_collection(params[:id], params[:api_key])
+      collection = validate_collection(sanitize_collection_name(params[:id]), params[:api_key])
```

Comment:
Currently the routes file uses ':id' when it is only a collection route (e.g. catalog/:id) and ':collectionId' is used when routing to items or documents (e.g. catalog/:collectionId/:itemId), so this case used :id since it is just a collection, no items/docs. 
To make it more consistent the routes file would need to be changed to make them consistent, which would also involve changing quite a few views and controllers aswell.